modules/distr-tracing-tempo-install-gateway-read-permissions.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-gateway-read-permissions_{context}"]
+= Configuring the read permissions for tenants
+
+You can onfigure the read permissions for tenants from the *Administrator* view of the web console or command line.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a cluster administrator with the `cluster-admin` role.
+* For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Define the tenants by adding the `tenantName` and `tenantId` parameters with your values of choice to the `TempoStack` custom resource (CR):
++
+.Tenant example in a `TempoStack` CR
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: redmetrics
+spec:
+...
+  tenants:
+    mode: openshift 
+    authentication: 
+      - tenantName: dev # <1>
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <2>
+...
+----
+<1> A `tenantName` value of the user's choice.
+<2> A `tenantId` value of the user's choice.
```

Comment:
@rubenvp8510, can you confirm that this step and the YAML block are correct?

Reply:
It is correct.